oslabs-beta/Kubiquity
# Add scripts

WHAT DOES THIS PR DO? 
1. Adds various scripts to our package.json

HOW TO TEST THIS PR:
1. Pull this PR using the GH CLI. 
2. Run `npm install`. 
3. Run `npm run dev` and verify that the FE and BE start, and that the Electron app opens. Because Electron will open before Webpack is done compiling the FE, you will have to refresh your code. 
4. Run `npm run frontend` amd verify that the FE starts and that the Electron app opens. Same issue as above. 
5. Run `npm run backend` and verify that nodemon runs our server. 
6. Run `npm run react` and verify that Webpack compiles our frontend (verify in terminal). 
7. Run `npm start` and verify that the Electron app starts. 

ADDITIONAL NOTES: 
- If you're working on the backend team and want/need to change the backend command (or any command), go ahead and do so. 
- I have no sense still of what exactly will be run once this app is in production, so that's something we should look into after our MVP presentation. 
- I cannot get `npm run build` to work—to be fixed at a different time, I suppose, since it's not blocking work.
